fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1304,7 +1304,13 @@
     /**
      * connection and socket timeout for broker client
      */
-    @ConfField()
+    @ConfField
     public static int broker_client_timeout_ms = 10000;
-}
 
+    /**
+     * Whether to collect routine load process metrics.
+     * Be careful to turn this on, because this will call kafka api to get the partition's latest offset.
+     */
+    @ConfField(mutable = true)
+    public static boolean collect_routine_load_lag_metrics = false;
```

Comment:
```suggestion
    public static boolean collect_routine_load_lag_metrics = false;
```
Better to name config starting with `enable_`

Reply:
done